tools/secure_notebook.py

```diff
@@ -90,12 +119,14 @@ def create_self_signed_cert(cert_dir, keyfile, certfiile):
 
     fullkey = os.path.join(location, keyfile)
     fullcrt = os.path.join(location, certfile)
+    with persist_config() as config:
+        config.NotebookApp.certfile = fullcrt
+        config.NotebookApp.keyfile = fullkey
 
-    config.NotebookApp.certfile = fullcrt
-    config.NotebookApp.keyfile = fullkey
-
-    with io.open(os.path.join(jupyter_config_dir(), 'jupyter_notebook_config.json'), 'w') as f:
-        f.write(six.u(json.dumps(config, indent=2)))
-
+    print('done.\n')
 
 
+if __name__ == '__main__':
+    print("This guide you into securing your notebook server.")
```

Comment:
I'd reword the msg as `This will guide you through the steps towards securing your notebook server.` Though a review by one of our native speakers, @takluyver or @minrk would be even better...